files/image_config/bannerconfig/banner-config.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -e
+
+STATE=$(sonic-cfggen -d -v 'BANNER_MESSAGE["MESSAGE"]["state"]')
+LOGIN=
+MOTD=
+LOGOUT=
+
+if [[ $STATE == "enabled" ]]; then
+    LOGIN=$(sonic-cfggen -d -v 'BANNER_MESSAGE["MESSAGE"]["login"]')
+    MOTD=$(sonic-cfggen -d -v 'BANNER_MESSAGE["MESSAGE"]["motd"]')
+    LOGOUT=$(sonic-cfggen -d -v 'BANNER_MESSAGE["MESSAGE"]["logout"]')
+
+    echo -e "$LOGIN" > /etc/issue.net
+    echo -e "$LOGIN" > /etc/issue
+    echo -e "$MOTD" > /etc/motd
+    echo -e "$LOGOUT" > /etc/logout_message
+fi
```

Comment:
If state is disabled, are you planning clear banner files?

Reply:
No, we do not clean banner messages.